store/contributions.js

```diff
@@ -1,5 +1,5 @@
-import { firebaseAction } from 'vuexfire'
 import { db } from '@/plugins/firebase'
+import { firebaseAction } from 'vuexfire'
```

Comment:
Revert the changes made in files not touched in this PR

Reply:
Okay